low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -1232,8 +1239,10 @@ if __name__ == "__main__":
         print('Missing arguments')
         parser.print_help()
         sys.exit(1)
-    storage_type = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{TYPE}", 'virtual').lower()
-    server_type = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{NODE_TYPE}", 'virtual').lower()
+    
+    response_dict = GlobalConf().fetch_global_config(["storage_type", "server_type"])
+    storage_type = response_dict['storage_type']
```

Comment:
Keep response_dict name shorter, like resp and refer elements directly as res['storage_type']